src/index.js

```diff
@@ -20,8 +39,19 @@ function App(props) {
     setIsAuthenticating(false);
   }
 
+  function handleDarkTheme() {
+    if (cook.get('thisCookie') === 'true') {
+      document.body.className = 'dark';
+      enableDarkMode();
+    } else {
+      document.body.className = 'light';
+      disableDarkMode();
+    }
+  }
+
   useEffect(() => {
     getAuthStatus();
+    handleDarkTheme();
```

Comment:
i see that we are doing a similar approach in the navbar as well. i suggest creating a file in [`src/APIFunctions`](https://github.com/SCE-Development/Core-v4/tree/dev/src/APIFunctions) called `darkTheme.js`. There u can define two functions for toggling and checking if the darktheme is active

```js
// put all the imports u need here

export function checkIfDarkThemeActive() {
  // paste ur code for checking
}

export function toggleDarkTheme() {
  // paste ur code for toggling
}
```

after u can just import these functions at the top of index.js and UserNavbar.js and call on them without duplicate code.

Reply:
1. you wanna call on `toggleDarkTheme` whenever u click on the switch.
1. in `index.js`, call on `checkIfDarkThemeActive` from `useEffect`

the navbar and index.js should contain no other dark theme logic, only the api functions. the api functions should take care of everything